specification/web/resource-manager/Microsoft.Web/stable/2022-09-01/StaticSites.json

```diff
@@ -3286,6 +3283,215 @@
           }
         }
       }
+    },
+    "/providers/Microsoft.Web/freeTrialStaticWebApps": {
+      "get": {
```

Comment:
Tenant level APIs need a sign off from the PAS team. Please send an email to SARBACRev, prasnal@microsoft.com to get a sign off

Reply:
Scheduled a security review. If they approve can we ignore these other validation issues. I sent a mail with the details.

Reply:
@rkmanda it seem like it will take some more time for us to get the security review through. Could we in the meantime see if there are any other changes required from my side or if we can get the requested exception? thanks